src/utils/links.tsx

```diff
@@ -1,5 +1,6 @@
 import GitBook from 'src/assets/icons/gitbook.svg';
 import SF from 'src/assets/icons/sf-logo-light.svg';
+import X from 'src/assets/icons/x.svg';
```

Comment:
`x.svg` is a icon for a close button.
Please set `x-twitter` using `@fortawesome/react-fontawesome`.